.travis.yml

```diff
@@ -1,6 +1,6 @@
 sudo: false
 language: node_js
 node_js:
-  - "8"
+  - "10"
```

Comment:
I think I’d keep it to node 8 as far as I know that’s Probot’s currently required version? It will prevent us from using Node 10 features that will break usage in Node 8